### ISSUE TITLE
Mystical Agriculture -> Secondary Seed Drops + TheOnePrope Error

### ISSUE THREAD

**Issue by tekook** [AllTheMods/ATM-9#117]
Don't really know if this is related to ATM or just MA.

(1) The secondary seed drops do not work currently. The chances are displayed but the seeds never drop.
(1.1) fertilizer gets drops corretly.

(2) Since like version 0.0.27 there is an error display when looking at a MA seed:
![2023-07-17_21 06 54](https://github.com/AllTheMods/ATM-9/assets/976407/bbb936a1-e47d-4474-892a-c3b381edde85)

The Server logs this error as long as you look at the seed:
[Server thread/ERROR] [ne.mi.fm.lo.RuntimeDistCleaner/DISTXFORM]: Attempted to load class net/minecraft/client/player/LocalPlayer for invalid dist DEDICATED_SERVER

Kind Regards

**Comment by Alucard1313** [comment]
Same here. The One Probe displays the same error, and when looking at any MA culture, the same flood of errors occurs in the console.
Here is the server log

[latest.log](https://github.com/AllTheMods/ATM-9/files/12073273/latest.log)

**Comment by Alucard1313** [comment]
I can confirm that the problem continues in modpack version 0.41.

**Comment by zastil** [comment]
I am also not getting seeds unless in a pot and not get 300% return using correct farmland for inferium seeds

**Comment by Niel0000** [comment]
Hello! My name is Niel, I'm a Brazilian ATM9 player. And this message is directed to the modpack devs, me and my brazilian friends reported some bugs. One of them is in the Xnet mod when we take a cable, or any block of the mod the FPS of the game drops! And with Integrated dynamics, the mechanical block squezer texture is weird. And the cards to use in the rftools builder are without craft. Well, if you have already been reported about these bugs, please disregard my message. Thank you very much for your attention.

**Comment by tekook** [comment]
this error got "fixed" in 0.043, since theoneprobe is no longer in the modpack.
Jade does not produce this error.

**Comment by tekook** [comment]
if anyone still wants secondary seed drops, this can be enabled via serverconfig/mysticalagriculture-common.toml.
```
[General] 
#Should crops have a chance of dropping a second seed when harvested?
	secondarySeedDrops = true
```
